changes/514.fix

```diff
@@ -0,0 +1,2 @@
+Fix `prepare()` not running when `start_session()` call hangs without raising Exception
+
```

Comment:
Please remove this extra trailing newline to prevent line-separation of PR number reference during auto-generation of changelogs at release.